packages/@aws-cdk/cloud-assembly-schema/lib/manifest.ts

```diff
@@ -21,6 +16,16 @@ const ASSEMBLY_SCHEMA = require('../schema/cloud-assembly.schema.json');
  */
 const SCHEMA_VERSION = require('../schema/cloud-assembly.version.json').version;
 
+/**
+ * VersionMismatchError error is thrown when there is CloudAssembly scheme mismatch between the CLI and the framework
+ */
+export class VersionMismatchError extends Error { 
+  constructor() {
+    super();
+    Object.setPrototypeOf(this, VersionMismatchError.prototype);
```

Comment:
Wut? Why this?

Reply:
This is before I realized that this is in fact a jsii module, so I can't use Custom Error. For the interest of conversation, this is required in TypeScript to enable `instanceOf  CustomError`: [extending-built-ins-like-error-array-and-map-may-no-longer-work](https://github.com/Microsoft/TypeScript/wiki/Breaking-Changes#extending-built-ins-like-error-array-and-map-may-no-longer-work)